frontend/src/pages/Admin/Components/Testimonial/Testimonial.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import style from "./testimonial.module.scss";
+import { AiFillEdit } from "react-icons/ai";
+import { AiOutlinePlus } from "react-icons/ai";
+
+export function Testimonial(props) {
+  return (
+    <div className={style["broadcast"]}>
+      <h1 style={{ textAlign: "center" }}> Testimonial </h1>
+      <div className={style["cards"]}>
+        <div className={style["card-item"]}>
+          <div className={style["clickable-card"]}>
+            <div className={style["card-title"]}>
+              <div onClick={() => props.setTab(21)} style={{ color: "white" }}>
+                CREATE TESTIMONIAL
+              </div>
+              <AiOutlinePlus className={style["add"]} />
+            </div>
+            <div className={style["card-content"]}>
+              <p style={{ textAlign: "left" }}>To Create a new Testimonial</p>
+              <p
+                // onClick={openEditor}
```

Comment:
please remove if not required

Reply:
I think it is necessary. To follow the code flow. I added it because in admin panel all cards with add or create property has this type of statement. so added it.